src/runtime/manifest.ts

```diff
@@ -93,3 +94,24 @@ export interface ManifestStack {
   requiredAPIs: ManifestRequiredAPI[];
   endpoints: Record<string, ManifestEndpoint>;
 }
+
+/**
+ * Returns the JSON representation of a ManifestStack, which has CEL
+ * expressions in its options as object types, with its expressions
+ * transformed into the actual CEL strings.
+ * @internal
+ */
+export function stackToWire(stack: ManifestStack): Object {
+  let wireStack = stack as any;
+  let traverse = function traverse(obj: Object) {
+    for (const [key, val] of Object.entries(obj)) {
+      if (val instanceof Expression) {
```

Comment:
if you rename `toCEL` as `toJSON` then `JSON.stringify` will call it for you automatically.